src/Row.ts

```diff
@@ -37,15 +37,44 @@ class Row extends RowBase<RowProperties> {
 				classes: this.classes(tableCss.table, css.rowTable)
 			}, [
 				v('tr', columns.map((column) => {
-					const { id, field } = column;
+					const { field, id } = column;
+
+					let value: any;
+					if (column.get) {
+						// Get the value from the column callback
+						value = column.get(item, column);
+					}
+					else if (!column.render) {
+						// Get the value using a property lookup on the item data
+						value = item.data[ field || id ];
+					}
+					else {
+						// This column has no value
+						value = '';
+					}
+
+					let content: DNode;
+					if (column.render) {
```

Comment:
`get` is specific to finding the value to be used in a cell (the combination of the item and column) and is meant to present an alternative to the default method of retrieving this value from the item based on a field.

`render` is meant to take this value and turn it into the UI that will be used inside the cell. `render` is reusable, whereas `get` is not. So `render` could appear across all columns, while `get` is unique to each column.

I made it so that `get` could return a `DNode` so that if someone really wanted to do everything in a single function, they could (though I agree it isn't great practice, it just seemed the best way to do things if we wanted to avoid having separate render methods for when you do and do not care about the cell's value).

I hope this is enough to explain why `get` and `render`, `value` and `content` are two separate ideas that seem to encourage separate levels of customization. If that can be agreed upon, I can remove the `DNode` return from `get` so it retains this clear separation and I guess I can also make the `render` function accept an object, though I wish there was a way to do this without so much overhead.